src/device/device_ledger.cpp

```diff
@@ -1479,20 +1489,31 @@ namespace hw {
             offset+=32;
           } else {
             // dummy: is_subaddress Aout Bout AKout
-            offset += 1+32*3;
+            offset += 2+32*3;
```

Comment:
Why is this change for ?

Reply:
one more byte for `is_change_address` in the `if` counterpart ,   comment was not updated...

Reply:
Hmm, OK, I don't see a code change to add that extra byte, that's what I was getting at.